modules/fedimint-mint-client/src/output.rs

```diff
@@ -105,6 +113,29 @@ pub struct MintOutputStatesCreated {
     pub(crate) issuance_request: NoteIssuanceRequest,
 }
 
+pub fn verify_map(
```

Comment:
**suggestion:** Comments, comments. I know it takes time to write, but as a reviewer i'm having much harder time reviewing without them and having to figure out what is what.